plugin/ochttp/trace.go

```diff
@@ -132,34 +132,39 @@ func (t *traceTransport) CancelRequest(req *http.Request) {
 	}
 }
 
-// NewHandler returns a http.Handler from the given handler
-// that is aware of the incoming request's span.
-// The span can be extracted from the incoming request in handler
-// functions from incoming request's context:
+// NewHandler returns a new handler.
+func NewHandler(base http.Handler) *Handler {
+	return &Handler{Handler: base, Propagation: defaultFormat}
+}
+
+// Handler is a http.Handler that is aware of the incoming request's span.
+// Use NewHandler to create new handlers.
+//
+// The extracted span can be accessed from the incoming request's
+// context.
 //
 //    span := trace.FromContext(r.Context())
 //
-// The span will be automatically ended by the handler.
+// The server span will be automatically ended at the end of ServeHTTP.
 //
 // Incoming propagation mechanism is determined by the given HTTP propagators.
-func NewHandler(base http.Handler, format propagation.HTTPFormat) http.Handler {
-	if format == nil {
-		format = defaultFormat
-	}
-	return &handler{handler: base, format: format}
-}
+type Handler struct {
+	// Propagation defines how traces are propagated. If unspecified,
+	// B3 propagation will be used.
```

Comment:
Where in the code do we fallback to B3 propagation if the zero value of Propagation is present?

Reply:
Done